lib/gitter.js

```diff
@@ -7,7 +7,15 @@ var FayeClient  = require('./faye.js');
 
 var Gitter = function(token, opts) {
   opts = opts || {};
+  
+  try {
+    token = token.replace(/\s+/g, '');
 
+    if (token.length < 40) throw new Error();
```

Comment:
Instead of catching an error, let's remove the the `try-catch` and `console.warn` directly inside the `if` statement